dts/bindings/clock/st,stm32mp13-pll-clock.yaml

```diff
@@ -28,6 +28,13 @@ description: |
   than 1000 MHz or program the mpuss_ck mux to use the MPUDIV
   (refer to the stm32mp13 reference manual for details)
 
+  div-m, div-p, div-q, div-r valid ranges are identicals for all 4 PLLs
+  mul-n valid range depends on the PLL
```

Comment:
I would use here the Name defined in stm32mp13 reference manual but also in formula above
```
DIVM1, DIVP, DIVQ and DIVR have identical valid ranges for all 4 PLLs.
DIVN valid range depends on the PLL
```
or use aka as below
```
div-m (aka DIVM1) , div-p (aka DIVP), div-q (aka DIVQ), div-r (aka DIVR)  have identical valid ranges for all 4 PLLs.
mul-n (aka DIVN) valid range depends on the PLL

```